hc/accounts/models.py

```diff
@@ -55,8 +56,21 @@ def set_api_key(self):
 
     def send_report(self):
         # reset next report date first:
+        if self.reports_frequency == 'Disabled':
+
+            self.reports_allowed = False
+            self.reports_frequency = 'Disabled'
+            return
+
+        if self.reports_frequency == 'Daily':
+            day = timedelta(seconds=2)
```

Comment:
Why 2 seconds?

Reply:
Had set it that way for demo purposes, I think I will set it back to the intended timelines.